docs/self-managed/console-deployment/configuration.md

```diff
@@ -9,14 +9,21 @@ description: "Read details on the configuration variables of Console Self-Manage
 Console Self-Managed is available only to [Enterprise customers](/reference/licenses.md#console).
 :::
 
-Console Self-Managed can be configured using environment variables and configuration parameters.
+Console can be configured using environment variables, configuration parameters, or a combination of both. When configuring your Console setup, keep in mind the following:
```

Comment:
This is probably a PM question, but do we want to emphasize "Console Self-Managed" as the component name? Or are they similar enough that we can drop "Self-Managed" with the right context?

Reply:
I think it is a good idea to specify self-managed as well, since it is a different development stream. It is quite different from SaaS.
cc @theburi